frontend/src/metabase/css/core/layout.css

```diff
@@ -5,6 +5,7 @@
   --xl-width: 1540px;
 }
 
+// When converting to styled components, use FullWidthContainer
```

Comment:
`//` is not a valid comment in css, it probably should be `/* When converting to styled components, use FullWidthContainer */`